CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
+## [v1.2.0] - 2024-10-10
+
+### Changed
+
+- [[DAS-2216](https://bugs.earthdata.nasa.gov/browse/DAS-2216)]
+  The Swath Projector has been updated with quick fixes to add support for TEMPO level 2 data.
```

Comment:
Sorry - one more nit pick: This change log would be better if it actually described what the changes added are. (The updates to the configuration file to correctly locate coordinate variables, and the optional transposing of the arrays based on dimension sizes)

If I was to read this change log, I wouldn't really know what this change does.

Reply:
Added more description in commit [6678d5a](https://github.com/nasa/harmony-swath-projector/pull/19/commits/6678d5a8f93fab5a79213d7ceb16a13538fb52c1)

Reply:
Awesome - looks good, thanks!